mass_editing/migrations/12.0.2.0.2/post-migration.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        "UPDATE mass_editing SET action_name = name WHERE action_name IS NULL"
```

Comment:
AFAIK this has to be done in a pre-migration, before the NOT NULL constraint is applied (as in https://github.com/OCA/server-ux/pull/221).

Reply:
That's not possible, as the column still doesn't exist. There's no problem though: next time you update, you will get the constraint.

Reply:
You may copy it, as done in the mentioned PR.

Reply:
OK, copy in pre-migration as in your PR.

Reply:
@pedrobaeza you seem to have deleted it instead. Could you check, please?

Reply:
Sorry, I didn't do `git add` of the new file. You can check now.